e2e/system_test.go

```diff
@@ -32,11 +33,81 @@ func generateAddressKeyPairs(num int, t *testing.T) []*addressKeyPair {
 	return pairs
 }
 
+// getAccountBalance is a helper method for fetching the Balance field of an account
+func getAccountBalance(
+	address types.Address,
+	rpcClient *jsonrpc.Client,
+	t *testing.T,
+) *big.Int {
+	accountBalance, err := rpcClient.Eth().GetBalance(
+		web3.Address(address),
+		web3.Latest,
+	)
+
+	assert.NoError(t, err)
+
+	return accountBalance
+}
+
+// getStakedBalance is a helper method for fetching the StakedBalance field of an account
+func getStakedBalance(
+	address types.Address,
+	rpcClient *jsonrpc.Client,
+	t *testing.T,
+) *big.Int {
+	var out string
+	if callErr := rpcClient.Call(
+		"stake_getStakedBalance",
+		&out,
+		web3.Address(address),
+		web3.Latest.String(),
+	); callErr != nil {
+		t.Fatalf("Unable to fetch staked balance")
+	}
+
+	stakedBalance, ok := new(big.Int).SetString(out[2:], 16)
+	if !ok {
+		t.Fatalf("Unable to convert staked balance response to big.Int")
+	}
+
+	return stakedBalance
+}
+
+var signer = crypto.NewEIP155Signer(100)
+
+type sendParams struct {
+	fromAddress types.Address
+	toAddress   types.Address
+	value       *big.Int
+	signingKey  *ecdsa.PrivateKey
+	t           *testing.T
+	rpcClient   *jsonrpc.Client
+}
+
+var defaultGasPrice = big.NewInt(1048576)
+
+func sendRawTransaction(params sendParams) (web3.Hash, error) {
+	// Create the transaction
+	txnObject := &types.Transaction{
+		From:     params.fromAddress,
+		To:       &(params.toAddress),
+		GasPrice: defaultGasPrice,
+		Gas:      1000000,
+		Value:    params.value,
+	}
+
+	signedTxn, err := signer.SignTx(txnObject, params.signingKey)
+	assert.NoError(params.t, err)
+
+	data := signedTxn.MarshalRLP()
+
+	// Do the transfer
+	return params.rpcClient.Eth().SendRawTransaction(data)
+}
```

Comment:
This method already exists in testserver.go L321. You can probably reuse that one? 🙂

Reply:
Got it, didn't catch it earlier.
Refactored to use the existing method 👍